docs/stdlib/numbers.rst

```diff
@@ -487,7 +506,7 @@ from :eql:type:`str` and :eql:type:`json`.
 
     :index: power pow
 
-    Power operation.
+    Performs a power (exponential) operation by two arbitrary numbers:
```

Comment:
```suggestion
    Produces the value of the left operand raised to the power of the right one.
```

Wanted to clarify this a bit. Took inspiration from [the Python docs](https://docs.python.org/3/library/functions.html#pow).